detox/e2e/support/server_api/common.js

```diff
@@ -2,7 +2,20 @@
 // See LICENSE.txt for license information.
 
 export const getResponseFromError = (err) => {
-    const {response: {data, status}} = err;
+    const {response} = err;
+    if (!response) {
+        const message = `No response from server at "${err.config.baseURL}".
+If testing against a server other than the default local instance, you may set the server URL via "SITE_URL" environment variable.
+`;
+
+        // Throw an error instead of failing silently
+        throw new Error(message);
```

Comment:
Throw an error and immediately failed when unreachable or no response from server.
<img width="947" alt="Screen Shot 2020-09-28 at 6 03 11 PM" src="https://user-images.githubusercontent.com/5334504/94421290-ed7d8000-01b7-11eb-99eb-9a2b5420fb5d.png">